task/04-arrays-tasks.js

```diff
@@ -286,7 +309,12 @@ function propagateItemsByPositionIndex(arr) {
  *   [ 10, 10, 10, 10 ] => [ 10, 10, 10 ]
  */
 function get3TopItems(arr) {
-   throw new Error('Not implemented');
+    function compareNumeric(a, b) {
+        if (a < b) return 1;
+        if (a > b) return -1;
+      }            
+    arr.sort(compareNumeric);
+    return arr.slice(0,3);
 }
```

Comment:
For array of numbers sort function can be simplified.   arr.sort((a,b) => a - b) - for ascending order,  arr.sort((a,b) => b - a) - for descending